src/build.py

```diff
@@ -902,24 +900,38 @@ def LLVM():
         # linking libtinfo dynamically causes problems on some linuxes,
         # https://github.com/emscripten-core/emsdk/issues/252
         '-DLLVM_ENABLE_TERMINFO=%d' % (not IsLinux()),
+        '-DCLANG_ENABLE_ARCMT=OFF',
+        '-DCLANG_ENABLE_STATIC_ANALYZER=OFF',
     ], build_dir)
 
+    if not IsMac():
+        # LLD isn't fully baked on mac yet.
+        command.append('-DLLVM_ENABLE_LLD=ON')
+
+    ninja_targets = ('all', 'install')
     if options.use_lto:
+        targets = ['clang', 'lld', 'llvm-ar', 'llvm-addr2line', 'llvm-cxxfilt',
+                   'llvm-dwarfdump', 'llvm-dwp', 'llvm-nm', 'llvm-objcopy',
+                   'llvm-objdump', 'llvm-ranlib', 'llvm-readobj', 'llvm-size',
+                   'llvm-strings', 'llvm-symbolizer']
```

Comment:
What do we use addr2line, cxxfilt, readobj, size, strings, and symbolizer for?

And until we remove llc from emscripten, I think we still need it?

Reply:
these are all just the standard binutils utilities, which sometimes have general usefulness for toolchain users and IMO just generally belong in a toolchain. We should add llvm-strip too, once I get around to fixing it. (addr2line is just a symlink to symbolizer).
I added 'opt' and 'llc' on a separate line to make them easier to remove later whenever we're ready.

Reply:
I see, thanks. And yes, I missed the other line with opt and llc...